### ISSUE TITLE
Searching: In group by Sender, file updates from non-relevant users will re-expand folders from Results section

### ISSUE THREAD

**Issue by Searinox** [zlatinb/muwire#129]
If you're searching and viewing results grouped by Sender, any new results that come in updating any user's files will trigger a re-expansion of the folder tree on the user you're viewing, even if no results from that user have been updated. A user's results could well be finished fetching, but if other users' results are still coming in, the folder view gets forcefully re-expanded. It doesn't make sense and it hampers trying to browse results because you have to wait 1-2 minutes for all the searching to complete before you can actually do anything.

**Comment by zlatinb** [comment]
I agree it's annoying, but it's not very easy to fix.  I'll need some time...

**Comment by zlatinb** [comment]
Ok, I think I got it to work at least a little better.  More specifically, results arriving from senders which are not in focus will not affect the rendering of the tree or table of the sender in focus.  However, new results from the same sender will collapse the tree.

Please give it a try, binary available [here](https://muwire.com/downloads/MuWire-0.8.12-beta6-GitHub129.zip)

**Comment by Searinox** [comment]
This works, and it makes a world of difference. I won't be submitting a bug on list updates of the focused sender causing collapse just yet as I do not believe it to be serious for this point in the app's development; if for whatever reason that situation changes, I will open an issue. Until then, this is resolved.
